docs/sql/scalar.txt

```diff
@@ -144,11 +144,82 @@ If the time zone is not specified, truncation is based on UTC time::
     +---------------+---------------+
     SELECT 3 rows in set (... sec)
 
+extract(field from source) returns long
```

Comment:
Would leave it here, most other docs refer to it as function too.

See 

- http://www.postgresql.org/docs/9.1/static/functions-datetime.html
- http://docs.oracle.com/cd/B19306_01/server.102/b14200/functions050.htm
- http://www.w3schools.com/sql/func_extract.asp